addons/website_sale_comparison/static/src/js/website_sale_comparison.js

```diff
@@ -36,6 +37,10 @@ var ProductComparison = publicWidget.Widget.extend(VariantMixin, {
      */
     start: function () {
         var self = this;
+        this.$scrollingElement = $().getScrollingElement();
+        this.$scrollingTarget = $().getScrollingTarget(this.$scrollingElement);
```

Comment:
`getScrollingElement` method is already adpated in vanilla from [v18](https://github.com/odoo/odoo/blob/18.0/addons/web/static/src/core/utils/scrolling.js#L176) add aTODO here to adapt the changes according to it, and try to avoid the use of jquery as much as possible.